### ISSUE TITLE
HDF5. File accessibility. Unable to open file.

### ISSUE THREAD

**Issue by hukai916** [GreenleafLab/ArchR#999]
I know similar issues have been reported before. This time, this occurs in **addBgdPeaks**.

Some suggest that setting "threads = 1" may solve the problem as for createArrowFiles, however, there is no "threads" argument in addBgdPeaks function. The problem occurs sporadically and I can't reproduce the error.

What I have tried:
1. rhdf5 version: rhdf5_2.36.0
2. h5 file is not corrupted as indicated by h5ls(getArrowFiles(my_archrproject)[1])
3. arrow file permissions are proper

**Comment by rcorces** [comment]
This has been an issue for a long time and still needs to be addressed.
See https://github.com/GreenleafLab/ArchR/issues/248 for the original report and one potential solution.
